scipy/optimize/tests/test_nonlin.py

```diff
@@ -141,7 +141,7 @@ def test_tol_norm_called(self):
 
         def local_norm_func(x):
             self._tol_norm_used = True
-            return np.absolute(x).max()
+            return abs(x).max()
```

Comment:
I'm not sure this change is needed. I think abs() is tested in test__root.py, so I think it is ok that this test keep using np.absolute().

Reply:
O. K.: 4d1d3a8